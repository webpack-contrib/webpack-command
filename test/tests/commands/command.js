const { test } = require('../../util');
const Command = require('../../../lib/commands/Command');

let command;

test('help command', module, () => {
  it(`should instantiate`, () => {
    command = new Command();
    expect(command).toBeInstanceOf(Command);
  });

  it(`should display help`, () => {
    const result = command.help();
    expect(result).toMatchSnapshot();
  });

  it(`should run`, () => {
    const stub = () => command.run({});
    expect(stub).not.toThrow();
  });
});
