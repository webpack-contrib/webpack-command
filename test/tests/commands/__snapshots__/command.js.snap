// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`help command > should display help #0 1`] = `"<base>"`;
